test/e2e/storage/volume_metrics.go

```diff
@@ -123,3 +175,22 @@ func getControllerStorageMetrics(ms metrics.ControllerManagerMetrics) map[string
 	}
 	return result
 }
+
+// Verifies the specified metrics are in `kubeletMetrics`
+func verifyVolumeStatMetric(metricKeyName string, namespace string, pvcName string, kubeletMetrics metrics.KubeletMetrics) {
+	found := false
+	if samples, ok := kubeletMetrics[metricKeyName]; ok {
+		for _, sample := range samples {
+			samplePVC, ok := sample.Metric["persistentvolumeclaim"]
+			Expect(ok).To(BeTrue(), "Error getting pvc for %s", metricKeyName)
```

Comment:
Do you want to attempt to process all samples in case more than one fails? Then call `Expect` if any sample fails?

Reply:
👍  - That would provide more information in case of failures - let me do that

Reply:
Don't see this change yet either...